README.md

```diff
@@ -78,16 +78,16 @@ fs.src('images/*.png')
 
 ## Troubleshooting
 
-On Mac OS X, there is an installation issue with `cairo` not being found. In this case, you can easily install it through homebrew:
+On Mac OS X, there is an installation issue with `cairo` not being found. In this case, you can easily install all [`canvas` module dependencies](https://github.com/Automattic/node-canvas#installation) through homebrew:
 
-```
-brew install cairo
+```bash
```

Comment:
Actually it`s aliases in github syntax highlighting. https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L3197